### DISCUSSION TITLE
Siri response language

### DISCUSSION THREAD

**Discussion by patgle** [milo526/homebridge-tuya-web#142] [Q&A] [answered]
Hello,
I'm not sure if this is the right place to ask, but is it possible to change the language of the Siri responses I receive after changing the color of my ceiling light?
Currently, the responses on my iPhone/iPad look something like this: "[Here you go], _light blue_!"  with the part in the [brackets] being in German (my language) and the _color_ in English. It's not a dramatic issue, but it would be nice to know if there is an option of changing the color term to German in order to understand what Siri says (her German-flavoured English is terrible ;) ) or if the source of the "problem" is Homebrige itself.

Happy holidays and greetings,
Patrick

**Comment by milo526** [comment, marked as answer]
Hey, this is a perfect place to ask this!

Sadly this is not possible, not even HomeBride can change this I believe. This is something that Apple controls through HomeKit.
The only "solution" here would be to switch Siri to English but that hardly seems like a solution at all.

Happy holidays!
MIlo

**Reply by patgle** [reply]
Thank you for your fast response and help, appreciate it ;)
